packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -1371,8 +1377,8 @@ contract OptimismPortalResourceFuzz_Test is CommonTest {
         // bounds to satisfy the assumptions listed in this specific section. These assumptions
         // serve only to act as an additional sanity check on top of the bounds and should not
         // result in an unnecessary number of test rejections.
-        vm.assume(gasLimit >= _gasLimit);
-        vm.assume(_minimumBaseFee < _maximumBaseFee);
+        _gasLimit = uint64(bound(_gasLimit, 0, gasLimit));
```

Comment:
We should keep these, they are already maintained above. Assumptions here are just sanity checks.

Reply:
fixed